File: home-loan-system/src/main/java/com/tripod/homeloansystem/models/Person.java
package com.tripod.homeloansystem.models;

import java.io.Serializable;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name = "person")
public class Person implements Serializable{

    private static final long serialVersionUID = 5934687059711010916L;

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "person_id")
    private Long personId;
    
    @Column(name = "first_name")
    private String firstName;

    @Column(name = "middle_name")
    private String middleName;

    @Column(name = "last_name")
    private String lastName;

    @Column(name = "username")
    private String username;

    @Column(name = "password")
    private String password;

    @Column(name = "email")
    private String email;

    @Column(name = "phone_number")
    private String phoneNumber;

    @Column(name = "address")
    private String address;

    @Column(name = "date_of_birth")
    private String dateOfBirth;

    @Column(name = "is_admin")
    private Boolean isAdmin;

    @Column(name = "is_first_login")
    private Boolean isFirstLogin;
    
}

File: hls-vue/src/models/person.js
class Person{
    constructor(personId, firstName, middleName, lastName, username, password, email, phoneNumber, address, dateOfBirth, isAdmin, isFirstLogin, authorities){
        this.personId = personId;
        this.firstName = firstName;
        this.middleName = middleName;
        this.lastName = lastName;
        this.username = username;
        this.password = password;
        this.email = email;
        this.phoneNumber = phoneNumber;
        this.address = address;
        this.dateOfBirth = dateOfBirth;
        this.isAdmin = isAdmin;
        this.isFirstLogin = isFirstLogin;
        this.authorities = authorities;
    }

}

/**
 * 
 * 
  {
    "personId":552,
    "firstName":"Charles",
    "middleName":"",
    "lastName":"Thomas",
    "username":"user",
    "password":"$2a$10$lTVNRR1UBPMwKmqxUtQe6e/9rlw6LtFC6JgKFr1BpsHbk2qK18Dpi",
    "email":"charles.thomas@example.com",
    "phoneNumber":"1234567890",
    "address":"BLR",
    "dateOfBirth":"01/01/1999",
    "isAdmin":false,
    "isFirstLogin":true,
    "authorities":[{"authority":"ROLE_USER"}],
    "accountNonExpired":true,
    "accountNonLocked":true,
    "credentialsNonExpired":true,
    "enabled":true
   }
 */